exercicios/para-casa/soraia/soraia.py

```diff
@@ -0,0 +1,213 @@
+dataset = {
+    ("Ana", "Silva"): {
+        "Turma": "Turma A",
+        "Notas": [7.5, 8.0, 9.0],
+        "Presença": [True, True, False, True, True],
+        "Participação": 8.5
+    },
+    ("Mariana", "Santos"): {
+        "Turma": "Turma B",
+        "Notas": [6.0, 7.5, 8.5],
+        "Presença": [True, True, True, False, True],
+        "Participação": 5.0
+    },
+    ("Carla", "Oliveira"): {
+        "Turma": "Turma A",
+        "Notas": [8.0, 7.5, 8.5],
+        "Presença": [True, True, True, False, False],
+        "Participação": 8.2
+    },
+    ("Juliana", "Ferreira"): {
+        "Turma": "Turma C",
+        "Notas": [9.0, 8.5, 7.0],
+        "Presença": [True, True, True, True, True],
+        "Participação": 8.7
+    },
+    ("Patrícia", "Souza"): {
+        "Turma": "Turma B",
+        "Notas": [5.0, 5.0, 5.0],
+        "Presença": [True, False, True, True, True],
+        "Participação": 7.2
+    },
+    ("Aline", "Martins"): {
+        "Turma": "Turma A",
+        "Notas": [8.5, 8.0, 9.0],
+        "Presença": [True, True, True, True, True],
+        "Participação": 8.5
+    },
+    ("Fernanda", "Costa"): {
+        "Turma": "Turma C",
+        "Notas": [6.5, 7.0, 8.0],
+        "Presença": [True, True, True, False, True],
+        "Participação": 7.2
+    },
+    ("Camila", "Pereira"): {
+        "Turma": "Turma B",
+        "Notas": [7.5, 8.0, 8.5],
+        "Presença": [True, True, True, True, True],
+        "Participação": 8.0
+    },
+    ("Luana", "Rodrigues"): {
+        "Turma": "Turma A",
+        "Notas": [9.0, 9.0, 8.5],
+        "Presença": [True, True, True, True, True],
+        "Participação": 8.8
+    },
+    ("Beatriz", "Lima"): {
+        "Turma": "Turma C",
+        "Notas": [8.0, 7.5, 7.0],
+        "Presença": [True, True, True, False, True],
+        "Participação": 7.5
+    }
+}
```

Comment:
O dicionário já estava definido e importado, essa parte não precisava ter alterado. Do mesmo modo que você usou os dados deste dicionário chamado dataset, você poderia ter usado usando esta notação:
O único ponto de atenção é que o arquivo precisa estar na mesma pasta.

**Sugestão:**
```suggestion
from dataset_alunas import dataset
```